owl/EASE-OBJ.owl

```diff
@@ -2760,12 +2848,6 @@ It is also possible that a Transient transitionsBack to the initial object. An e
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#DesignedContainer">
         <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#DesignedArtifact"/>
-        <rdfs:subClassOf>
```

Comment:
Well because the Containment disposition got removed, and I was not sure if being a designed container actually implies that you can insert something.
If we want to assume that, then I would re-add the axiom for the Insertion disposition for now.